.github/workflows/linting.yml

```
@@ -1,10 +1,15 @@
 name: Linting and code quality check
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
 
 jobs:
   lint:
-    uses: mundialis/github-workflows/.github/workflows/linting.yml@main
+    uses: mmacata/github-workflows/.github/workflows/linting.yml@ruff-config
```

```suggestion
    uses: mundialis/github-workflows/.github/workflows/linting.yml@main
```
Update when https://github.com/mundialis/github-workflows/pull/42 is merged.